src/shared/components/callout/callout.css

```diff
@@ -10,7 +10,7 @@
   border-top: var(--hw-border-thickness) solid var(--hw-color-primary-dark);
   padding: var(--hw-spacing-small-4) 0;
 
-  p {
+  p, div:nth-child(2) {
```

Comment:
This selector is dependent on a "hw-block" which is present in the Callout part in Enonic. However it is not required for the component. On Hedwig QA for example there is not any divs inside the callout.
We need to make sure the content, whatever it is, is grey while the H3 is black.

Reply:
This might have come out wrong. What I mean is that the selector is dependent on a "div" element - that is usually a hw-block. However using a div.hw-block is not required for the Callout component so we should not include a selector for it in the Callout css.

Reply:
This is the old outdated change. I'll highlight it in the below comment